frontend/src/components/activePage/TableInstance/RowInstanceActions/RowInstanceActionsDropdown.tsx

```diff
@@ -78,43 +90,65 @@ const RowInstanceActionsDropdown: FC<IRowInstanceActionsDropdownProps> = ({
     <Dropdown
       trigger={['click']}
       overlay={
-        <Menu onClick={click => dropdownHandler(click.key.toString())}>
+        <Menu
+          onClick={({ key }) => {
+            dropdownHandler(key.toString() as DropDownAction);
```

Comment:
```suggestion
            dropdownHandler(key as DropDownAction);
```
The toString shouldn't be needed.